### PR TITLE
Bump sqlite

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

File: S/SQLite/build_tarballs.jl [hunk 1]
```diff
@@ -7,7 +7,7 @@ version = v"3.31.1"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://sqlite.org/2020/sqlite-autoconf-3310100.tar.gz", "62284efebc05a76f909c580ffa5c008a7d22a1287285d68b7825a2b6b51949ae"),
+    ArchiveSource("https://sqlite.org/2020/sqlite-autoconf-3320300.tar.gz", "a31507123c1c2e3a210afec19525fd7b5bb1e19a6a34ae5b998fbd7302568b66"),
     ArchiveSource("https://git.archlinux.org/svntogit/packages.git/plain/trunk/license.txt?h=packages/sqlite&id=33cad63ddb1ba86b7c5a47430c98083ce2b4d86b",
```
Comment (giordano) [inline S/SQLite/build_tarballs.jl:11]: This should be a `FileSource`:
```suggestion
    FileSource("https://git.archlinux.org/svntogit/packages.git/plain/trunk/license.txt?h=packages/sqlite&id=33cad63ddb1ba86b7c5a47430c98083ce2b4d86b",
```
No idea how it worked before

**Comment by giordano** [comment]
Note that this doesn't fix https://github.com/JuliaDatabases/SQLite.jl/issues/214, the symbol is still missing.  We probably need to enable it with some flags.  This is the build script in ArchLinux: https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/sqlite&id=d8b6ba561152179e943807054388462b7259e6df

**Comment by giordano** [comment]
By using the same flags as ArchLinux now I get:
```
% nm libsqlite3.so.0.8.6|grep sqlite3_column_table_name
0000000000045920 T sqlite3_column_table_name
0000000000045930 T sqlite3_column_table_name16
```
which should effectively solve https://github.com/JuliaDatabases/SQLite.jl/issues/214

**Comment by quinnj** [comment]
Yay, thanks!

**Pull request merged**